### ISSUE TITLE
unable to read custom_values from Preferences object

### ISSUE THREAD

**Issue by varun43in** [ruckus/quickbooks-ruby#473]
Hi, I'm trying to get a list of custom values.  Here's my code

```
@items = service.query("Select * from Preferences")

<%= @items.entries[0].sales_forms.custom_fields %>

```
But this throws a noMethodError.  Any ideas?

Thanks for your help!

**Comment by ruckus** [comment]
Hmm, what version are you running? This is working for me.

**Comment by varun43in** [comment]
0.6.6

In my Gemfile:
gem "quickbooks-ruby", '0.6.6'

Gemfile.lock shows:
quickbooks-ruby (0.6.6)

**Comment by ruckus** [comment]
Doh, I see what happened. I merged in a PR with support for Preferences/CustomFields but it never got  released.

I just released `0.6.7` -- please give RubyGems time to update and then give that a shot.

**Comment by varun43in** [comment]
It works now.  Thanks a lot!
